libyara/modules/console/console.c

```diff
@@ -0,0 +1,265 @@
+/*
+Copyright (c) 2021. Wesley Shields <wxs@atarininja.org>. All Rights Reserved.
```

Comment:
Let's use the unified copyright header `Copyright (c) 2021. The YARA Authors. All Rights Reserved.` You are already listed in the `AUTHORS` file.